packages/canvas/canvas-sprite/src/CanvasSpriteRenderer.ts

```diff
@@ -131,9 +135,9 @@ export class CanvasSpriteRenderer
 
         if (sprite.tint !== 0xFFFFFF)
         {
-            if (sprite._cachedTint !== sprite.tint || sprite._tintedCanvas.tintId !== sprite._texture._updateID)
+            if ((sprite as any)._cachedTint !== sprite.tint || sprite._tintedCanvas.tintId !== sprite._texture._updateID)
```

Comment:
In these cases, I think I'd rather make `_cachedTint` an internal public property (still private from the docs). Makes this a lot less brittle.

Reply:
@bigtimebuddy how do you suggest to do it? Because it's not possible to extend Sprite (incorrectly extends interface Sprite property _cachedTint is protected and can't be public in another interface).

Reply:
Looks like @Zyie did this already in #6665, if you want to see for reference. Given the choice of removing the access modifier (i.e., `protected`) or casting as `any`, it's much better to do the former.